README.md

```diff
@@ -207,6 +207,35 @@ used for purposes such as logging, where multiple crates provide similar functio
 - Property-based Testing - [proptest](https://crates.io/crates/proptest)
 - Temporary Files and Directories - [tempfile](https://crates.io/crates/tempfile)
 
+## Docker
+An environment that includes a local [MinIO](https://min.io/) instance and an edge node using the [MinIO](https://min.io/)
+server as the remote object store, can be set up using [Docker](https://docs.docker.com/). Note that since 
+[Rust](https://www.rust-lang.org/) is a compiled language and a more dynamic `modelardbd` configuration might be needed,
+it is not recommended to use the [Docker](https://docs.docker.com/) environment during active development of `modelardbd`. 
+It is however ideal to use during more static deployments or when developing components that utilize `modelardbd`.
+
+Downloading [Docker Desktop](https://docs.docker.com/desktop/) is recommended to make maintenance of the created 
+containers easier. Once [Docker](https://docs.docker.com/) is set up, navigate to the `ModelarDB-RS` folder.
+The [MinIO](https://min.io/) server can be created by running the services defined in 
+[docker-compose-minio.yml](docker-compose-minio.yml). The services can be built and started using the following command.
+
+```shell
+docker-compose -p modelardata-minio -f docker-compose-minio.yml up
+```
+
+After the [MinIO](https://min.io/) server service is created, a [MinIO](https://min.io/) client is created to initialize
+the development bucket `modelardata`, if it does not already exist. To browse the created [MinIO](https://min.io/) blob
+store, a graphical user interface can be used. Open http://localhost:9001 to view it in the browser. The default username
+and password, `minioadmin`, can be used to log in. Note that the [MinIO](https://min.io/) server is handled in a separate
+compose file in case a similar [MinIO](https://min.io/) server instance already exists.
+
+Similarly, the `modelardbd` instance itself can be built and started from the same folder using the following command.
+The started instance can be accessed from `grpc://127.0.0.1:9999`.
+
+```shell
+docker-compose -p modelardbd up              
```

Comment:
I think it is standard to have a "base" compose file that contains the services that should always be started and then add a suffix to other optional compose files. I have left it as it was for now but it might be changed in the future when more services are added.

Reply:
Ok